__tests__/nylas-api-spec.js

```diff
@@ -211,6 +211,17 @@ describe('Nylas', () => {
       );
     });
 
+    test('should use a provider when provided in the options', () => {
+      const options = {
+        loginHint: 'ben@nylas.com',
+        redirectURI: 'https://localhost/callback',
+        provider: 'icloud',
+      };
+      expect(Nylas.urlForAuthentication(options)).toEqual(
```

Comment:
nit - For tests like this, I'd recommend using string composition in the expected value to make sure your tests don't break if you need to change your options in the future.

eg.
```
 const loginHint = 'ben@nylas.com',
     redirectURI = 'https://localhost/callback',
     provider = 'icloud';

 const options = {
      loginHint,
      redirectURI,
      provider,
 };
 
 expect(Nylas.urlForAuthentication(options))
     .toEqual(`https://api.nylas.com/oauth/authorize?client_id=newId&response_type=code&login_hint=${loginHint}&redirect_uri=${redirectURI}&provider=${provider}`
```

Reply:
Thanks! Updated the test cases to follow this format!